README.md

```diff
@@ -70,3 +70,13 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
 
 # tsb-jahresbericht-2021
+
+## Fonts
+
+The fonts used in this project are not public. As a member of Technologiestiftung you can access them via
+
+Google Drive: \_CityLAB Berlin/Visuals/06 Design/tsg-standard-font Clan\
+or\
+Server Space: LWS$/Öfentlichkeitsarbeit/Jahresberichte/21_Jahresbericht\
```

Comment:
typo `Öffentlichkeitsarbeit` and they are actually currently not there :-D

Also I think they are somewhere else already on that drive but I don't know where :-D 
@marcoka2000 maybe you know?

Reply:
ah true @marcoka2000 told me where they are i mixed it up. Will take care of it.